bazaarvoice/emodb
# EMO-7697 New Postman tests are written for next endpoints:

New Postman tests for next endpoints have been added

DELETE /bus/1/{subscription}
GET /bus/1/{subscription}
GET /bus/1/{subscription}/claimcount
GET /bus/1/{subscription}/peek
GET /bus/1/{subscription}/poll
GET /bus/1/{subscription}/size

## What Are We Doing Here?

Here is where you should describe the problem you are solving as it relates to the Github Issue referred above, adding any fine
details on the solution that might otherwise not be recognizable for someone
unfamiliar with the changes. Add some pictures if it helps.

## How to Test and Verify

Check out this PR
Start emodb on local
Open postman
Import "emodb_tests.postman_collection.json" as a collection
Import "sor-1-_table.postman_environment.json" as environment variables
Execute tests in postman
Profit
Expected result: All tests are successfully executed on local environment

## Risk
Low
There are no changes to application code base

## Level
All newly added postman tests should be executed without failures.

## Required Testing
Smoke, Regression, or Manual. (All changes except documentation need smoke
testing at a minimum).

## Risk Summary
There are no risks to application, only tests are involved and it is about adding a new tests

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
